src/components/operations/operation-details/ko/runtime/authorization.ts

```diff
@@ -85,6 +87,8 @@ export class Authorization {
         this.selectedSubscriptionKey.subscribe(this.applySubscriptionKey.bind(this));
         this.selectedGrantType.subscribe(this.onGrantTypeChange);
         this.selectedSubscriptionKey(null);
+        this.selectedAuthorizationServer(this.authorizationServers() ? this.authorizationServers()[0] : null);
```

Comment:
Sorry, I did not explain my question.
I wanted to ask why do we want to call `this.authorizationServers()` 2 times, instead of one? If the first response is not null, then we will invoke method **again** for no reason (i dont think response will change between 2 invocations)
```
this.selectedAuthorizationServer(this.authorizationServers() ? this.authorizationServers()[0] : null);
```

Reply:
hey @DeagleGross , it is not a method. It is knockout js, the way to call of the variable is like this. it simply is this.authorizationServers in C#.